### PR TITLE
add a blog (wxbot-admin)

### PR DESCRIPTION
**Pull request by skyvow**
Fixes # .

Make sure these boxes are checked before creating your pull request -- thank you!

- [ ] Add Blog Header, including title, author, date.  
    For example:  

    ``` md
    ---
    title: "Your Great Blog Title"
    author: your_github_id
    image: /assets/2020/08-your-blog-slug/teaser.jpg
    ---

    YOUR_FIRST_SENTENCE_SHOULD_BE_A_SUMMARY_FOR_YOUR_POST
    ```

- [ ] Keep all filenames & url as lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor` instead of `2017-10-06-WeChat PC Impactor`,
- [ ] Embed the photo & video before publishing, save all external file to a folder named by `${MONTH}-${YOUR_BLOG_SLUG}/` under `/assets/2020` directory.
- [ ] Add teaser image for the blog
- [ ] Add your self as an author to the author data file at <https://github.com/wechaty/wechaty.js.org/blob/master/_developers/YOUR_GITHUB_USERNAME.md> with your GitHub username. And please also remember to reference yourself with the Github username in your blog post, so that we can have nice author information besides your blog after we published it!
  For excample:
  ```md
  ---
  name: Jiarui LI (李佳芮)
  site: https://lijiarui.github.io
  bio: 句子互动创始人 & CEO，微软人工智能最具价值专家 (AI MVP)
  avatar: /assets/developers/lijiarui/avatar.png
  email: rui@juzi.bot
  twitter: https://weibo.com/u/2175505900
  ---
  ```
  Here is the example file:
  - lijiarui.md: https://github.com/wechaty/wechaty.github.io/blob/master/_developers/lijiarui.md
  - It will shown like this on wechaty blog: https://wechaty.github.io/developers/lijiarui/

To read more about the contribute guideline, visit: <https://github.com/wechaty/wechaty.js.org#contribute-guideline>

### CODE REVIEW

**Comment by skyvow** [comment]
@huan @lijiarui Please help review it.

**Comment by huan** [comment]
@skyvow thanks for your great blog post pull requests!

Please wait @lijiarui to review but she's quite busy those weeks.

Thank you very much for your patient.

File: jekyll/_posts/2020-10-27-wxbot-admin.md [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+  - vue
+  - wechaty
+  - wechaty-puppet-padplus
+  - nuxt
+  - feature
+image: /assets/2020/wxbot-admin/screenshot.png
+---
+
+> 作者: [Skyvow](https://github.com/skyvow)
+> Code: [Github](https://github.com/wux-weapp/wxbot)
```
Comment (wj-Mcat) [inline jekyll/_posts/2020-10-27-wxbot-admin.md]: These will be the short description of blog, Please use a short description to display in a better way. refer to: [commandline-filter](https://github.com/wechaty/wechaty.js.org/blame/master/jekyll/_posts/2020-08-28-commandline-filter.md#L7) Thanks a lot.

**Review by wj-Mcat — CHANGES_REQUESTED** [review]
To better demonstrate your blog, please complete the following changes.

**Review by wj-Mcat — APPROVED** [review]
LGTM

**Comment by skyvow** [comment]
@wj-Mcat @lijiarui @huan 大佬们，啥时候能帮忙 merge 啊，token 过期都快一个月了，对开发者来说耐心都要没了，很难支持呀。。。

**Review by lijiarui — APPROVED** [review]
LGTM

**Comment by lijiarui** [comment]
Congratulations! 🎉 🎉 🎉

You become a wechaty contributor successfully! 👏 👏 👏

Please:

- [ ] Update your profile here: [Contributor List](https://docs.google.com/spreadsheets/d/1XcDoIczyIclqXP1p90Sz7S0n4Q22xjFzJmjPFlU2g1E/edit#gid=0), then other contributors will know you better!
- [ ]  Join wechaty org by clicking https://github.com/wechaty
- [ ]  Add my wechat: ruirui_0914 and send me this pr link. I will invite you into Wechaty Contributor Room

**Pull request merged**